unified-installer/astra-unified-installer.sh

```diff
@@ -2531,7 +2559,7 @@ step_generate_torc_patch() {
             if [ -z "$_EXISTING_TRIDENT_OPERATOR_PULL_SECRETS" ]; then
                 torc_patch_list+='{"op":"replace","path":"/spec/imagePullSecrets","value":['"$IMAGE_PULL_SECRET"']},'
             else
-                torc_patch_list+='{"op":"add","path":"/spec/imagePullSecrets/-","value":"'"$IMAGE_PULL_SECRET"'"},'
```

Comment:
Those double quotes are needed since the value is a JSON string. It's a little convoluted, but the jist of it is:
1. The first double quotes `"` are taken literally because they are within a set of single quotes `'`
2. The single quote `'` after this is closing the set of single quotes that was started at `torc_patch_list+='`
3. The second double quotes `"` are bash quotes to prevent globbing/word splitting (see https://github.com/koalaman/shellcheck/wiki/SC2086) from when `$IMAGE_PULL_SECRET` gets expanded.

Reply:
Ohhhhh, got it. Ill fix that.

Reply:
Fixed, good catch. Thanks